config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
-    get "static_pages/home"
-    get "static_pages/help"
+    root "static_pages#home"
+    get "help" => "static_pages#help"
```

Comment:
`get "help" => "static_pages#help"` thành `get "/help", to:  "static_pages#help"`

fix tương tự bên dưới